src/Components/SavedSearchAlert/Components/AddFiltersScreenRarity.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react"
+import { Box, Checkbox, Flex, Spacer, Text } from "@artsy/palette"
+import { useSavedSearchAlertContext } from "Components/SavedSearchAlert/SavedSearchAlertContext"
+
+export const rarityValues = [
+  {
+    name: "Unique",
+    value: "unique",
+  },
+  {
+    name: "Limited Edition",
+    value: "limited edition",
+  },
+]
```

Comment:
**note**: We're still discussing these options and it might actually be premature to leave out "Open Edition" and "Unknown Edition" (my fault). Need to bring this up with other colleagues. That rarity filter overall I think could use an update to be more useful/intuitive.